web/docusaurus.config.js

```diff
@@ -71,7 +71,7 @@ async function createConfig() {
   /// END CUSTOM CODE ///
   return {
     title: 'VAST',
-    tagline: 'Actionable insights at your fingertips.',
+    tagline: 'The open-source security pipelines and storage engine.',
```

Comment:
```suggestion
    tagline: 'The open-source pipelines and storage engine for security.',
```

We should also change the description of the GitHub repository alongside this.